### ISSUE TITLE
generic binary name pp

### ISSUE THREAD

**Issue by olafhering** [ocaml-ppx/ppxlib#224]
The binary `pp` conflicts with a binary of the same name from another, unrelated package:
````
found conflict of ocaml-ppxlib-0.21.0-12.2.x86_64 with perl-PAR-Packer-1.052-1.2.x86_64
  /usr/bin/pp
````

Unless `pp` is sort of an ABI because other tools outside of ppxlib depend on it, please consider renaming `pp`.

**Comment by stedolan** [comment]
I've just run into the same issue (various test binaries with generic names being installed), but it looks like it's already been fixed by @pitag-ha in #223.

**Comment by pitag-ha** [comment]
Indeed, the binary `pp` was a test binary that accidentally got installed in 0.22. It's fixed now in 0.23. Sorry for that!
